WooCommerce/Classes/Authentication/Navigation Exceptions/RoleErrorViewController.swift

```diff
@@ -80,6 +80,15 @@ class RoleErrorViewController: UIViewController {
 
     func configureLinkButton() {
         linkButton.applyLinkButtonStyle()
+
+        // enable multi line for accessibility support
+        linkButton.titleLabel?.textAlignment = .center
+        linkButton.titleLabel?.adjustsFontForContentSizeCategory = true
+        linkButton.titleLabel?.lineBreakMode = .byWordWrapping
+        if let linkTitleLabel = linkButton.titleLabel {
+            linkButton.pinSubviewToAllEdgeMargins(linkTitleLabel)
+        }
```

Comment:
We have a private method `enableMultipleLines()` method on `UIButton+Helper` to help with these situations.

Maybe it's worth mixing it with the `applyLinkButtonStyle` to avoid duplicating that logic.

Something like
```swift
private func applyLinkButtonStyle(enableMultipleLine: Bool = false) {
    // code
}
```

Reply:
Sure!